docs/source/guide/install.md

```diff
@@ -110,17 +110,12 @@ If you want to use nightly builds or extend the functionality, consider download
 ```bash
 git clone https://github.com/heartexlabs/label-studio.git
 cd label-studio
-python setup.py develop
+# Install all package dependencies
+pip install -e .
+# Start the server in development mode at http://localhost:8000
```

Comment:
It will start at http://localhost:8080

Reply:
also we need to run DB migrations
```
python label_studio/manage.py migrate
```
before runserver

Reply:
Updated README and install doc with this feedback